docs/source/commandline/commandline-all.rst

```diff
@@ -1931,33 +1931,63 @@ Core Input Options
 
 **-joystick_map** *<map>* / **-joymap** *<map>*
 
-    Controls how joystick values map to digital joystick controls. MAME accepts
-    all joystick input from the system as analog data.  For true analog
-    joysticks, this needs to be mapped down to the usual 4-way or 8-way digital
-    joystick values.  To do this, MAME divides the analog range into a 9x9 grid.
-    It then takes the joystick axis position (for X and Y axes only), maps it to
-    this grid, and then looks up a translation from a joystick map.  This
+    Controls how joystick values map to digital joystick controls.   
+    
+    Systems such as Pac-Man use a 4-way joystick and will actually exhibit
+    undesired behavior when a diagonal is triggered; in the case of Pac-Man,
+    movement will stop completely at intersections when diagonals are triggered
+    and the game will be considerably harder to play correctly.  Many other
+    arcade cabinets used 4-way or 8-way joysticks (as opposed to full analog
+    joysticks), so for PC-style true analog joysticks this then needs to be
+    mapped down to the expected 4-way or 8-way digital joystick values.  This
+    is because MAME accepts all joystick input from your controller as analog
+    data.
```

Comment:
I don't particularly like the wording here.  "PC-style true analog joysticks" may have been a thing back when the 15-pin game port was in use, but these days there isn't really such a thing as a "PC-style joystick" at all.  Say something like "…so for true analog joysticks, including flight sticks and analog thumb sticks,…" or something.  Also the last sentence is pointless.  MAME doesn't accept *all* joystick input as analog data – it accepts analog axis values as analog data, and saying so is redundant.

Reply:
Corrected.